app/models/ems_cluster.rb

```diff
@@ -232,11 +232,11 @@ def all_resource_pools_with_default
 
   # Parent relationship methods
   def parent_folder
```

Comment:
Unless we can name this "feature", it probably is better to duplicate the methods.